### ISSUE TITLE
ISIS hold timer for a neighbor is always ~60sec irrespective of the neighbor's hold time

### ISSUE THREAD

**Issue by DivyaBharathiS** [rtbrick/bngblaster#97]
Bug Description:

ISIS adjacency exists between a Router (R1) and a Bngblaster. On R1, hold timer is configured as 180sec. And from blaster it is expected that it announces the neighbor to be down after 180sec. But it is observed that it declares the neighbor to be down in approximately  60sec. 

Version (`bngblaster -v`): Version: 0.7.9

JSON configuration:

```{
    "interfaces": {
        "tx-interval": 1,
        "rx-interval": 1,
        "io-slots": 4096,
        "network": [
            {
                "interface": "SN-14-S4",
                "address": "131.0.0.2/24",
                "gateway": "131.0.0.1",
                "isis-instance-id": 1,
                "isis-level": 2
            }
        ]
    },
    "isis": [
        {
            "instance-id": 1,
            "area": [
                "49.0001/24"
            ],
            "system-id": "1921.6800.1007",
            "router-id": "192.168.1.7",
            "hostname": "BBL1",
            "hello-padding": true,
            "sr-base": 70000,
            "sr-range": 2000,
            "sr-node-sid": 7002,
            "teardown-time": 15,
            "lsp-tx-interval": 5,
            "lsp-tx-window-size": 5
                    }
                ]
            }

```

1. Configure 180sec hold time on R1
2. Make the ISIS neighbor down on R1

Expected behavior:

Bngblaster is expected to declare the neighbor to be down in 180sec

Bngblaster log for isis:
```
Jul 05 16:13:49.797809 ISIS L2 adjacency UP on interface SN-14-S4 
Jul 05 16:13:50.488119 Resolve network interfaces
Jul 05 16:13:50.488205 All network interfaces resolved
Jul 05 16:16:50.206916 ISIS RX L2-LSP PDU checksum error on interface SN-14-S4
Jul 05 16:17:49.502686 ISIS P2P-Hello timeout on interface SN-14-S4
Jul 05 16:17:49.502735 ISIS L2 adjacency DOWN on interface SN-14-S4 
Jul 05 16:21:20.883952 Received signal Interrupt (2), initiating teardown
```

**Comment by GIC-de** [comment]
Fix merged to `dev`!

**Comment by GIC-de** [comment]
Fixed in 0.7.10!
